client/client.go

```diff
@@ -0,0 +1,64 @@
+package client
```

Comment:
It seems like `client/client.go` and `client/config.go` should be combined since both files are fairly short and tightly coupled.

Also, we will be adding the equivalent client support for mixer configuration soon to the manager repo. It might be good to rename this package `proxy` and then we'd end up with the following directory/package layout, e.g.

```
client/proxy.go         // package proxy
client/proxy_test.go  
client/mixer.go         // package mixer
client/mixer_test.go  
```